components/dashboard/src/prebuilds/list/PrebuildList.tsx

```diff
@@ -140,10 +157,28 @@ const PrebuildsListPage: FC = () => {
                         onLoadNextPage={() => fetchNextPage()}
                         onFilterChange={handleFilterChange}
                         onSort={handleSort}
+                        onTriggerPrebuild={() => setShowRunPrebuildModal(true)}
                     />
                 )}
 
-                {!showTable && !isLoading && <PrebuildListEmptyState />}
+                {showRunPrebuildModal && (
+                    <RunPrebuildModal
+                        onClose={() => setShowRunPrebuildModal(false)}
+                        onRun={() => {
+                            const queryKey = getListConfigurationsPrebuildsQueryKey(organization.data?.id ?? "", {
```

Comment:
Sorry review this change a little bit late, but we could `refetch` this query they should work the same.

```
    const {
        refetch,
    } = useListOrganizationPrebuildsQuery()
```

Reply:
Sometimes, `refetch` will not make the new prebuild row shown, because it may not the first page or sorted that new prebuild is not included in this result.

Maybe most common case is to, add a new row manually in current view? Not blocking

Reply:
I think re-fetching sounds fine. Let's see it if causes issues down the road.